assets/css/admin.scss

```diff
@@ -81,7 +85,7 @@ promote-job-modal {
 		padding: 0px 25px 25px;
 	}
 }
-dialog {
+.promoteJobsDialog {
```

Comment:
In this one, since it's a class, we could use the format `.promote-jobs-dialog`. ;)

Reply:
Fixed in acafdbb6aee91bd404fd30cfc7e34803509c4389